Centaurus.Domain/Quanta/Handlers/AlphaQuantumHandler.cs

```diff
@@ -102,6 +104,8 @@ void ProcessQuantum(MessageEnvelope envelope)
                     Status = ClientExceptionHelper.GetExceptionStatusCode(exc),
                     OriginalMessage = envelope
                 });
+
+                QuantumHandleFailed(envelope, exc);
```

Comment:
Misleading name for callback. Please rename to something more straightforward, like `QuantumFailed`